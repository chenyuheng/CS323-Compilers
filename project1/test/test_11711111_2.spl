int testla() {
    int test_var_1 = 0;
    int b = 9;
    int c;
    c = b test_var_1;

