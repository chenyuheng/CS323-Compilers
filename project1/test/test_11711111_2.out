Error type B at Line 5: Missing semicolon ';'
Error type B at Line 1: Missing closing curly brace '}'
